### DISCUSSION TITLE
How to hide bus stops?

### DISCUSSION THREAD

**Discussion by sryze** [osmdroid/osmdroid#1900] [Q&A] [answered]
I'd like to hide the default bus stop icons because I'm drawing my own bus stops. I could't find a way to hide map features. Is there a way to do this with osmdroid?

**Comment by monsieurtanuki** [comment, marked as answer]
Hi @sryze!
No it's not possible. We don't draw bus stop icons, we only draw pre-processed map tiles where the bus stops are already displayed.
That said, you can play with different tile providers: one of them may not display bus stops. Have a look at [`TileSourceFactory`](https://github.com/osmdroid/osmdroid/blob/master/osmdroid-android/src/main/java/org/osmdroid/tileprovider/tilesource/TileSourceFactory.java) .
